### ISSUE TITLE
3.0.0-beta.3: Infinite render invalidation detected

### ISSUE THREAD

**Issue by maxfierke** [sir-dunxalot/ember-tooltips#273]
Upon leaving a route with various tooltips, this error was thrown when running tooltip's `dispose` in a runloop on `willDestroyElement`

Stacktrace:
```
Uncaught Error: infinite rendering invalidation detected
    at Array.loopEnd (renderer.js:138)
    at Backburner._trigger (backburner.js:889)
    at Backburner.end (backburner.js:475)
    at Backburner._run (backburner.js:833)
    at Backburner.run (backburner.js:520)
    at Object.run (ember-metal.js:4405)
    at Class.willDestroyElement (ember-tooltip-base.js:203)
    at Class.superWrapper [as willDestroyElement] (ember-utils.js:418)
    at Class.trigger (core_view.js:62)
    at Class.superWrapper [as trigger] (ember-utils.js:418)
```

Ember version: 2.18.2
ember-tooltips version: 3.0.0-beta.3

**Comment by maxfierke** [comment]
@sir-dunxalot this one is definitely a release-blocker for 3.0.0

**Comment by backspace** [comment]
I’m trying out the beta with `travis-web`, which is on Ember 3.4, and we’re getting that error too, in [our tests](https://travis-ci.org/travis-ci/travis-web/builds/432552928#L621). 😯

**Comment by maxfierke** [comment]
Fixed in `3.0.0-beta.4`
